_episodes/01-conda-tutorial.md

```diff
@@ -0,0 +1,61 @@
+---
+
+title: "Getting Started with Conda"
+teaching: 15
+exercises: 0
+questions:
+- "What is Conda?"
+- "How do I install Conda?" 
+objectives:
+- Get conda installed!
+keypoints:
+- conda can be installed in two ways (Anaconda and Miniconda)
+- conda package manager works across systems
+
+---
+
+### What is Conda?
+[**Conda**](http://conda.pydata.org/docs/) is an **open source `package` and `environment` management system for any programming languages, but very popular among python community,** for installing multiple versions of software packages, their dependencies and switching easily between them. It works on Linux, OS X and Windows.
+
+[This Continuum blog post is a terrific, recent and comprehensive introduction to conda](http://www.continuum.io/blog/conda-data-science) targeted to data scientists. It also has links to a presentation (Youtube and slides) on the same material. An extra nifty aspect of this material is that it "explores how to use conda in a multi-language data science project" with an example combining Python and R libraries.
+
+### Installing Miniconda
+
+##### Windows
+Click [here](http://conda.pydata.org/miniconda.html) to download the proper installer for your Windows platform (32 or 64 bits).
+We recommend to download the Python 3 version of Miniconda. You can still create Python 2 environments using the Python 3 version of Miniconda.
+
+When installing, you will be asked if you wish to make the Anaconda Python your default Python for Windows.
+If you do not have any other installation that is a good option.  If you want to keep multiple versions of python on your machine (e.g. ESRI-supplied python, or both 32 and 64 bit versions of Anaconda), then don't select the option to modify your path or modify your windows registry settings.
```

Comment:
Do people use 32 bit versions these days? If not, maybe we can remove this example, to reduce the complexity.